hatchways/team-eagle
# FE : Voting

In this pull request, I:
- Made the following changes to client:
  - Made URL entering not force redirect you to home page when logged in
  - Created poll show page
  - Made poll images (in `Dashboard`) into an individual customizable component called 
  `PollImages`

- Made the following changes to server:
  - Changed `voteIds` property to `numVotes` where it only displays an integer rather than an array
  - Added `timeStamps` to `VoteSchema`

#### How to test:
1. `yarn install`
2. Start app and go to `http://localhost:3000/polls/test` and see how it looks like

(Should be responsive)


PS: Do you think that it would be wiser to create local context for the poll page (rather than 3-level deep prop passing)?
